cmd/consts.go

```diff
@@ -0,0 +1,6 @@
+package cmd
+
+const (
```

Comment:
Why do we need a separate `consts.go` file? Please just add to the consts declared in `root.go`. They should be available to all other cmds since they're in the same package.